e3db-crypto-android/src/main/java/com/tozny/e3db/crypto/KeyStoreManager.java

```diff
@@ -0,0 +1,186 @@
+package com.tozny.e3db.crypto;
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * 
+ * Copyright (c) 2018 
+ * 
+ * All rights reserved.
+ * 
+ * e3db-java
+ * 
+ * Created by Lilli Szafranski on 4/6/18.
+ * 
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.os.Build;
+import android.security.keystore.KeyPermanentlyInvalidatedException;
+import android.support.v4.hardware.fingerprint.FingerprintManagerCompat;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Since we explicitly run `checkArgs()` to verify arguments aren't null at runtime, do we still need the extra annotation?

Reply:
Removed.